src/Microsoft.ApplicationInsights/Microsoft.ApplicationInsights.csproj

```diff
@@ -75,5 +79,11 @@
     </AdditionalFiles>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json">
```

Comment:
Is Microsoft.Extensions.Configuration.Json needed only for netstand 1.3 and net46? what about net45? is it missing by accident?

Reply:
if its required for all targets, then simply remove Conditions

Reply:
I believe this was me wandering off on a tangent earlier on in my change. I will clean it up (we don't need this reference at all).